core/db/migrate/20160225152313_remove_tax_rate_from_shipping_rate.rb

```diff
@@ -0,0 +1,29 @@
+class RemoveTaxRateFromShippingRate < ActiveRecord::Migration
+  def up
+    Spree::ShippingRate.find_each do |shipping_rate|
```

Comment:
Hm, I do have to call `compute_amount` on the `tax_rates`, which is a method with very weird behaviour that's actually about to change. I just had a long chat with @tvdeyen about ways to remedy this particular rabbit hole (it's deep) and essentially it will mean moving the weird behaviour of `compute_amount` into the `Spree::Calculator::DefaultTax`, then marking that class as deprecated, and creating a new, simpler `Spree::Calculator::Tax::Default` which hopefully makes things better. This is not ideal, but we have to make sure changing the implementation of how taxes are calculated does not break old taxes.